src/client/public/css/components/separator.css

```diff
@@ -2,3 +2,9 @@
 	border: none;
 	border-top: 1px solid rgba(156, 156, 156, 0.68);
 }
+
+@media (max-width: 480px) {
+	.separator {
+		display: none;
+	}
+}
```

Comment:
we do the separator to be hidden everywhere when the page width is less than 480px